polynote-kernel/src/main/scala/polynote/kernel/remote/DeploySparkSubmit.scala

```diff
@@ -70,18 +71,19 @@ object DeploySparkSubmit extends DeployCommand {
       sparkArgs ++ Seq(jarLocation) ++ serverArgs
   }
 
-  override def apply(serverAddress: InetSocketAddress): RIO[Config with CurrentNotebook, Seq[String]] = for {
+  override def apply(serverAddress: InetSocketAddress, classPath: Seq[Path]): RIO[Config with CurrentNotebook, Seq[String]] = for {
     config   <- Config.access
     nbConfig <- CurrentNotebook.config
     path     <- CurrentNotebook.path
   } yield build(
     config,
     nbConfig,
     path,
+    classPath.map(_.toUri.toURL),
     serverArgs =
       "--address" :: serverAddress.getAddress.getHostAddress ::
       "--port" :: serverAddress.getPort.toString ::
-      "--kernelFactory" :: classOf[LocalSparkKernelFactory].getName ::
+      "--kernelFactory" :: "polynote.kernel.LocalSparkKernelFactory" ::
```

Comment:
Why this change?

Reply:
I moved `DeploySparkSubmit` to `polynote-kernel`, so it can't see the `LocalSparkKernelFactory` class anymore. I did that basically so that `Main` doesn't have to be in `polynote-spark` anymore.